From 72f9493792dab0f9eb35c09c5a8a64bb27d144a0 Mon Sep 17 00:00:00 2001
From: Filip Pytloun <filip@pytloun.cz>
Date: Fri, 12 Aug 2016 19:51:25 +0200
Subject: Do not use setuptools_scm

---
 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index 32371f2..1629291 100755
--- a/setup.py
+++ b/setup.py
@@ -43,8 +43,6 @@ setup(
     install_requires=requirements,
     extras_require=extra_requirements,
     tests_require=test_requirements,
-    setup_requires=['setuptools_scm'],  # not needed when using packages from PyPI
-    use_scm_version={'write_to': 'khal/version.py'},
     classifiers=[
         "Development Status :: 4 - Beta",
         "License :: OSI Approved :: MIT License",
